### DISCUSSION TITLE
Maintainers hacktoberfest call

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#5468] [Community Call]
## AGENDA

1. Topic ideas
2. Availability
3. EddieHub repos ready
4. Actions

### 1. TOPIC IDEAS

**Discord**
- Playwright for NASA
- Cypress/Playwright for Kubernetes 
- Finder (Cahl)   
   - Tests (auth)
   - Removed blocked users
- Office hours (help) x4
   - Guests to help?
- Speed networking

- After hacktoberfest: show case / proud of PRs

**Live streams**
- Finder (Cahl)
   - Intro to project (auth, graphql), Docs (quickstart, tests, ...)
   - Adding skeleton tests / Existing skeleton tests
   - Tests with auth (Cahl + Debbie)? Possibly do the same for LinkFree?
- LinkFree (`nextjs` branch)
   - Accessibility (Emma) - encourage others to raise issues + fix
   - Tests (playwright after Finder)
- EddieBot (Naomi)
- Project search/reviews (submitted, search GitHub) (Emma?)
   - GitHub Actions for commands
   - Accessibility 
- NASA/Kubernetes? (after Discord call)

**Twitter Space**
- Hacktoberfest challenges (are you stuck?)
- 

**Video**
- Hacktobefest show case / proud of PRs

### 2. Availability

**Pre:**
- [x] 25 September: Twitter Space 4pm 
- [x] 26 September: Livestream 9am 
- [ ] 30 September: Twitter Space 9am
- [x] 30 September: Livestream w/ Francesco (intro into open source, fork, pull request)

**Week 1** 
- [ ] 1st October: Live stream with client?
- [x] 7 October: Livestream with Dan Jutan 6pm 
- [ ] 9 October: Twitter Space 4pm

**Week 2**
- [ ] 10 October: Livestream 9am
- [ ] 12 October: Twitter Space 9am
- [ ] 14 October: Livestream 4pm
- [ ] 16 October: Twitter Space 4pm

**Week 3**
- [ ] 17 October: Livestream 9am
- [ ] 19 October: Twitter Space 9am
- [ ] 21 October: Livestream 4pm
- [ ] 23 October: Twitter Space 4pm

**Week 4**
- [ ] 24 October: Livestream 9am
- [ ] 26 October: Twitter Space 9am
- [ ] 28 October: Livestream 4pm
- [ ] 30 October: Twitter Space 4pm

- [ ] 31 October: Livestream Time: ?

### 4. Actions

- Eddie: message in `team Eddie` for `hacktoberfest-practice` maintainers
- Eddie: check DiscordBotNetworking still works
- Sara: Office hours x2 per week (early/late uk time)
   - Guests: EddieHub maintainers, for example Kyle, Emma, Cahl...

**Comment by eddiejaoude** [comment]
IDEA 1: community video

All say the same sentence and the video will be edited to use 1 word from each person (thanks @EmmaDawsonDev)

If you want to get involved here are the details...

- FORMAT: video to be recorded in **horizontal** at the highest possible quality
- SENTENCE: `Hacktoberfest is coming! You can get involved too. Open Source is for everyone. Come and geek out with us in the  the most inclusive open source community EddieHub. We are all learning together, community first, code second!`
- DEADLINE: **10pm UK time Tuesday 27th of September**
- EDITING: do **not** edit the file, my video editor will do all the editing, so if you make a mistake, just pause and start the sentence again
- UPLOAD: upload videos below 👇

**Recording tips**
- have the light (window) in front of you, just like the camera
- talk louder than normal for the microphone to record it better
- play back the recording to see if it looks and sounds ok

**Reply by naomi-lgbt** [reply]
https://user-images.githubusercontent.com/63889819/192150521-2b89d160-74ec-41e8-a035-cafea8344cd1.mp4

Hope this works!

**Reply by LR-TechX** [reply]
I couldn't do it in the horizontal 😅😂

**Reply by EmmaDawsonDev** [reply]
Here's mine:



https://user-images.githubusercontent.com/57045550/192348411-82c03816-94f7-434a-ad98-6b7d4b9fb33f.mp4

**Reply by Pradumnasaraf** [reply]
Here's mine:

https://user-images.githubusercontent.com/51878265/192527775-2bc23cd5-ea22-42d8-8db4-2027bd0974a0.mov

**Reply by kumarsonsoff3** [reply]
Okay, cool! 
I got a bit late, but here it is!! 😁

https://user-images.githubusercontent.com/88786642/192602783-e9286d13-e223-448e-bbf5-26fd96fe1c2d.mp4

**Reply by eddiejaoude** [reply]
thank you 👍

**Comment by tamalCodes** [comment]
Here's mine : 

https://user-images.githubusercontent.com/72851613/192486876-32e9be02-14ca-4668-826a-eef036b0eede.mp4

**Reply by eddiejaoude** [reply]
thank you 👍

**Comment by Aniket1026** [comment]
Mine is here :


https://user-images.githubusercontent.com/92184189/192585307-f9cc67de-e039-42e3-9975-d1a2e5d6a4b8.mp4

**Reply by eddiejaoude** [reply]
thank you 👍
